tensorboard/plugins/mesh/metadata_test.py

```diff
@@ -26,6 +26,13 @@
 from tensorboard.util import test_util
 
 
+class FakeResult(object):
```

Comment:
Why do you need a `FakeResult` class instead of just constructing the
actual metadata? You can change

```python
result = FakeResult(version=metadata.get_current_version(), components=0)
```

to

```python
result = plugin_data_pb2.MeshPluginData(
    version=metadata.get_current_version(),
    components=0,
)
```

and drop the class entirely.

Reply:
done